readthedocs/projects/tasks/search.py

```diff
@@ -56,6 +57,45 @@ def fileify(version_pk, commit, build, search_ranking, search_ignore):
         log.exception('Failed during ImportedFile syncing')
 
 
+@app.task(queue="web")
+def sync_downloadable_artifacts(
+    version_pk, commit, build, artifacts_found_for_download
+):
+    """
+    Create ImportedFile objects for downloadable files.
```

Comment:
I'm not sure why this task is part of the search tasks. I understand it's not related with search itself, but more with the build process, in my opinion. So, I would probably move it to `projects.tasks.utils` or similar.